assets/tests/components/map.test.tsx

```diff
@@ -78,272 +69,55 @@ const vehicle: Vehicle = {
   endOfTripType: "another_trip",
   blockWaivers: [],
 }
-const trainVehicle: TrainVehicle = {
-  id: "red1",
-  latitude: 42.24615,
-  longitude: -71.00369,
-  bearing: 15,
-}
 
 describe("map", () => {
-  test("renders", () => {
```

Comment:
Is there a reason not to keep a basic render test?

Reply:
Yeah. We need the test file to use the real Leaflet implementation for other tests,`react-test-renderer` can't handle the real implementation and needs a mock, and I couldn't figure out how to unmock Leaflet for only part of the test file, it had to be all or nothing.

If we wanted to keep the snapshots, one thing we could try is having a separate test file that only does the snapshots, so that file could use the mocked Leaflet while this file doesn't. But the snapshots were so shallow, and the `draws x` tests do render everything, so I figured it wasn't worth it.

Reply:
Yeah, makes sense.